src/core/lv_group.c

```diff
@@ -97,6 +97,16 @@ lv_group_t * lv_group_get_default(void)
     return default_group;
 }
 
+lv_group_t * lv_group_get_child_group(lv_obj_t * obj)
```

Comment:
I think this API name may confuse the Micropython bindings generator. (cc @amirgon)

Reply:
Good point. I think this can be moved to lv_obj.

Reply:
👍